test/stores/env-test.js

```diff
@@ -19,5 +19,15 @@ vows.describe('nconf/stores/env').addBatch({
       assert.lengthOf(env.whitelist, 0);
       assert.equal(env.separator, '');
     }
+  },
+  "An instance of nconf.Env with readOnly option set to false": {
+    topic: new nconf.Env({readOnly: false}),
+    "should have it's readOnly property set to false": function (env) {
```

Comment:
@louis-murray in this test, you should also assert that calling `.set()` actually changes the value.

Additionally, it would be good to modify an existing `readOnly` test to ensure that calling `.set()` fails to modify a value.